docs/zh/latest/how-to-build.md

```diff
@@ -21,36 +21,37 @@ title: 如何构建 Apache APISIX
 #
 -->
 
-## 步骤1：安装依赖
-
-Apache APISIX 的运行环境需要依赖 NGINX 和 etcd，所以在安装 Apache APISIX 前，请根据您使用的操作系统安装对应的依赖。我们提供了 **CentOS7** 、**Fedora 31 & 32** 、**Ubuntu 16.04 & 18.04** 、 **Debian 9 & 10** 和 **MacOS** 上的依赖安装操作步骤，详情请参考 [安装依赖](install-dependencies.md)。
-
-通过 Docker 或 Helm Chart 安装 Apache APISIX 时，已经包含了所需的 NGINX 和 etcd，请参照各自对应的文档。
-
-## 步骤2：安装 Apache APISIX
+## 步骤1：安装 Apache APISIX
 
 你可以通过 RPM 仓库、RPM 包、Docker、Helm Chart、源码包等多种方式来安装 Apache APISIX。请在以下选项中选择其中一种执行。
 
 ### 通过 RPM 仓库安装（CentOS 7）
 
-这种安装方式适用于 CentOS 7 操作系统。Apache APISIX 已经支持适用于 CentOS 7 的 RPM 仓库。请运行以下命令安装 RPM 仓库和 Apache APISIX。
+这种安装方式适用于 CentOS 7 操作系统。
```

Comment:
Users may ask: how about other versions of CentOS?

Reply:
we currently only offer RPMs for Centos 7. In fact, AFAIK, Centos 7 is very common, Centos 6 and 8 are less common.